Header header
wbc_ros/TaskConfig config
float32 activation
uint8 timeout
float32[] weights
float32[] y_ref
float32[] y_solution
float32[] y
